charts/px-backup/templates/px-backup/pxcentral-etcd.yaml

```diff
@@ -168,7 +121,9 @@ data:
 
     # Constants
     AUTH_OPTIONS=""
-    export ETCDCTL_ENDPOINTS="pxc-backup-etcd-0.pxc-backup-etcd-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:2380,pxc-backup-etcd-1.pxc-backup-etcd-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:2380,pxc-backup-etcd-2.pxc-backup-etcd-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:2380"
+    ETCDCTL_ENDPOINTS="http://pxc-backup-etcd-0.pxc-backup-etcd-headless:2379,http://pxc-backup-etcd-1.pxc-backup-etcd-headless:2379,http://pxc-backup-etcd-2.pxc-backup-etcd-headless:2379,"
```

Comment:
should the port be 2380? also there is a "," at the end.

Reply:
removed